misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -689,6 +748,35 @@ abstract class PathAssertionsSpec(
             expect(folder).isDirectoryFun()
         }
     }
+
+    describeFun(hasSameBinaryContentAs.name) {
```

Comment:
I suggest you describe hasSameBinaryContentAs and hasSameTextualContentAs in the same describeFun
```suggestion
    describeFun(hasSameBinaryContentAs.name, hasSameTextualContentAs.name) {
```